components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -394,6 +394,13 @@ Rules of type `Node` can have conditions with attributes `Chef Organization`, `C
 Navigate to the project list page once more. Select `Update Projects`. Upon completion of the update, you should be able to
 filter by `project-devops` across Automate's dashboards and see only the ingested data that you expect.
 
+#### Effortless Infra Project
+To create a project that contains all Effortless Infra nodes, add a node ingest rule with a condition that uses an attribute of Chef Server and a value of "localhost".
+
+![](/images/docs/effortless-project-rule.png)
+
+The above rule matches on a node's Chef Server field, which is set to "localhost". This works because all Effortless Infra nodes do not have a Chef Server. Then, by adding a second condition that matches a specific Chef Policy name, subgroups (i.e., nodes matching a specified policy name) of Effortless Infra nodes can be created. 
```

Comment:
@lancewf To clarify, to create a project that contains all Effortless Infra nodes, you need 1) the condition shown in the included screenshot, and 2) a second condition that matches a specific Chef Policy name specific to the user's set-up? Or is including the second condition optional?

Reply:
to get all effortless nodes you need a condition to match _localhost_ in the chef server attribute, then if you want to scope that down, even more, you could add additional conditions.